assets/app/views/settings.html

```diff
@@ -0,0 +1,83 @@
+      <div ng-if="(quotas | hashSize) == 0"><em>{{emptyMessageQuotas}}</em></div>
+      <table ng-if="(quotas | hashSize) > 0" class="table">
+        <thead>
+          <th>Resource type</th>
+          <th>Used</th>
+          <th>Max</th>
+        </thead>
+        <tbody ng-repeat="quota in quotas">
+          <tr ng-if='!quota.status.used' class="danger">
+            <td colspan="5">
+              <span class="pficon-layered" data-toggle="tooltip" data-placement="right" title="Missing quota status" style="cursor: help;">
+                <span class="pficon pficon-error-octagon"></span>
+                <span class="pficon pficon-error-exclamation"></span>
+              </span>
+              Status has not been reported on this quota usage record.  Any resources limited by this quota record can not be allocated.
+            </td>
+          </tr>
+          <tr ng-class="{warning: quota.status.used && quota.status.used[resourceType] >= quota.status.hard[resourceType]}" ng-repeat="(resourceType, specMax) in quota.spec.hard">
+            <!-- TODO prettify resourceType? -->
+            <td>
+              <span ng-if="quota.status.used && quota.status.used[resourceType] >= quota.status.hard[resourceType]" class="pficon-layered" data-toggle="tooltip" data-placement="right" title="Quota limit reached" style="cursor: help;">
+                <span class="pficon pficon-warning-triangle"></span>
+                <span class="pficon pficon-warning-exclamation"></span>
+              </span>
+              {{resourceType}}
+            </td>
+            <td>
+              <span ng-if="!quota.status.used">&mdash;</span>
+              <span ng-if="quota.status.used">{{quota.status.used[resourceType] | usageWithUnits : resourceType}}</span>
+            </td>
+            <td>
+              <span ng-if="!quota.status.hard">{{specMax  | usageWithUnits : resourceType}}</span>
+              <span ng-if="quota.status.hard">{{quota.status.hard[resourceType]  | usageWithUnits : resourceType}}</span>
+            </td>
+          </tr>
+        </tbody>
+      </table>
+      <h2>
+        <span>Resource limits</span>
+        <a href="javascript:;" style="margin-left: 5px; cursor: pointer;" class="small pficon pficon-help" data-container="body" data-toggle="popover" data-placement="right" data-trigger="focus" data-content="Resource limits define minimum and maximum values that can be created for each resource type."></a>
+      </h2>      
+      <!-- TODO probably need to explain what this is -->
+      <div ng-if="(limitRanges | hashSize) == 0"><em>{{emptyMessageLimitRanges}}</em></div>
+      <table ng-if="(limitRanges | hashSize) > 0" class="table">
+        <thead>
+          <th>Resource type</th>
+          <th>Min</th>
+          <th>Max</th>
+        </thead>
+        <tbody ng-repeat="limitRange in limitRanges">
+          <tr ng-repeat-start="limit in limitRange.spec.limits"></tr>
+          <tr ng-repeat="(type, min) in limit.min">
```

Comment:
if min and max are not guaranteed to have the same set of resources, iterate over max to make sure we surface all max limits